content/posts/downstream-automation.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Downstream automation is here"
```

Comment:
Thank you for writing this down!

Is this meant to be a "heads-up" to our users or a guide that we'll update and refer to in future?

Reply:
I would say `heads-up`, but now I am not sure what's better...;)

It's true that documentation of separate jobs is not enough to see the whole workflow.

Reply:
My point is: we need a proper guide, very user-friendly, to describe in detail how to bring upstream releases into Fedora (using Packit) :)

Reply:
I see it can act as one. So, we can either convert this to that guide or make a slightly changed version of this that will be preserved/updated and linked in the future. I don't like duplicates, but in this case, I would rather create a separate guide.

Reply:
yup, sounds good, let's keep it and once we start working on the guide, we can get inspired here and even change/update it (for reference packit/packit.dev#351)